crt0stack/Cargo.toml

```diff
@@ -23,3 +23,4 @@ keywords = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+enumerate = { path = "../enumerate" }
```

Comment:
Which means, it's not a standalone crate anymore, which can be published independently. :disappointed:

Reply:
Why can't we just publish `enumerate`?

Reply:
yay! Do it! I can use it in a lot of places :)